pkg/kademlia/kademlia_test.go

```diff
@@ -61,6 +61,7 @@ func TestNewKademlia(t *testing.T) {
 		assert.NotNil(t, actual.nodeClient)
 		assert.NotNil(t, actual.routingTable)
 	}
+	assert.NoError(t, os.RemoveAll("db"))
```

Comment:
It should create the test directory in a temp directory.

Reply:
db is a temp directory -- would you prefer a different name ? maybe tempdb ?

Reply:
I mean inside `os.TempDir`

Reply:
Oh ok yea that makes sense!